packages/preact/test/hooks/useSwatches.spec.ts

```diff
@@ -194,4 +194,17 @@ describe("useSwatches", () => {
       )
     ])
   })
+
+  it("should return the correct selectedSku when all options match", () => {
```

Comment:
add more tests
how does the return value look like if only a single dimension is chosen?